<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Usuário entrando</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(253, 254, 232);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>110</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 233, 19);
border-radius: 15px;
border-style: outset;
border-width: 0px;</string>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>380</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 163, 2);
border-radius: 10px;
border-style: outset;
border-width: 0px;</string>
    </property>
    <property name="text">
     <string>Cadastre-se agora!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>380</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(114, 75, 17);</string>
    </property>
    <property name="text">
     <string>Não está cadastrado?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>280</y>
      <width>222</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>110</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Perfil.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>175</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Chave.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>175</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>400</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>by antthLuca</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
